socorro/lib/ooid.py

```diff
@@ -95,3 +99,27 @@ def dateFromOoid(ooid):
 
     """
     return dateAndDepthFromOoid(ooid)[0]
+
+
+CRASH_ID_RE = re.compile(r"""
+    ^
+    [a-f0-9]{8}-
+    [a-f0-9]{4}-
+    [a-f0-9]{4}-
+    [a-f0-9]{4}-
+    [a-f0-9]{6}
+    [0-9]{6}      # date in YYMMDD
+    $
+""", re.VERBOSE)
+
+
+def is_crash_id_valid(crash_id):
+    """Returns whether this is a valid crash id
+
+    :arg str crash_id: the crash id in question
+    :arg boolean strict: whether or not to be strict about the throttle character
+
+    :returns: True if it's valid, False if not
+
+    """
+    return bool(CRASH_ID_RE.match(crash_id))
```

Comment:
I want to be testing crash ids in other places, too, so I added this lib function.